dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/SnowFlakeUtilsTest.java

```diff
@@ -23,7 +23,8 @@
     @Test
     public void testNextId() {
         try {
-            for (int i = 0; i < 5; i++) {
+            for (int i = 0; i < 100; i++) {
+                Thread.sleep(1);
```

Comment:
Is there any reason for us to use `sleep` here?
```suggestion
```

Reply:
This is a UT test to see if a duplicate is generated in the next millisecond